src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlight.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.RetryRule;
+import bio.terra.stairway.RetryRuleExponentialBackoff;
+import bio.terra.workspace.service.buffer.BufferService;
+import org.springframework.context.ApplicationContext;
+
+// NOTE: DO NOT USE. Currently just a shell class to exercise connection to Buffer Service.
+public class CreateGoogleContextRBSFlight extends Flight {
+  public CreateGoogleContextRBSFlight(FlightMap inputParameters, Object applicationContext) {
+    super(inputParameters, applicationContext);
+
+    ApplicationContext appContext = (ApplicationContext) applicationContext;
+    BufferService bufferService = appContext.getBean(BufferService.class);
+    RetryRule retryRule =
+        new RetryRuleExponentialBackoff(
```

Comment:
I can think of few use cases we want to retry:

1. Pool is empty, RBS is working on creating new resources(you get 404 error code)
2. Other errors like 500, 503. 
The current retryRule might be good for 2. but we need longer wait time for 1. Creating a resource takes 10-15 minutes. 16 seconds is not enough for RBS to refill a pool. 
Another option is a fixedRule like retry every minutes for few time?  That is what [RBS chooses](https://github.com/DataBiosphere/terra-resource-buffer/blob/12afba8a634b9eb1109b668ee7b30d4e4e9fa314/src/main/java/bio/terra/buffer/service/resource/flight/StepUtils.java#L19)

Reply:
Got it. That actually might make exponential make sense. But it does seem like we want to design the Flight so that we have one step to ensure the pool is up and active and then another to handout the resource. The first should have a shorter retry as those errors are more likely to either be transient or totally fatal, while the other should have a longer retry period to allow for project creation, and ideally some logging along the way.